firmware/thruna/BMS/Src/App/states/App_AllStates.c

```diff
@@ -11,7 +11,7 @@
 #define CELL_FULLY_DERATED_TEMP (60.0f)
 
 // Num of cycles for voltage and cell temperature values to settle
-#define NUM_CYCLES_TO_SETTLE (3U)
+#define NUM_CYCLES_TO_SETTLE (30U)
```

Comment:
Why did we change this?

Reply:
Was seeing over voltage fault on startup, 30 may be excessive but 3 was too little. Perhaps more time should be taken to tune this and check if a FSM missing hb ignore timer is actually needed. I obviously won’t be around but if the car is together with the accumulator in, would be nice to check these things over

Reply:
ok fair enough. don't love that we need to extend this all of a sudden. 
maybe drop a TODO and we can look into reducing this later.

Reply:
It could be that it’s been silently faulting for a while now, only noticed it while debugging a issue with the latched fault signal going to the DIM. Ended up being completely unrelated, but dipping into fault state and setting the BMS ok status low should be enough to latch the fault in hardware and cause the DIM to display a latched fault. 

My suspicion is that this was actually happening at comp as well, and that it bounced out of fault state and set the ok status back high before hardware was able to latch the fault, though I have no way to prove that.

Definitely should add a TODO: and investigate further